app/jobs/whitehall_document_import_job.rb

```diff
@@ -15,10 +15,19 @@ class WhitehallDocumentImportJob < ApplicationJob
   def perform(document_import)
     document_import = WhitehallImporter::Import.call(document_import)
     document_import = WhitehallImporter::Sync.call(document_import)
+
+    if document_import.completed?
+      GdsApi.whitehall_export.document_migrated(document_import.whitehall_document_id)
```

Comment:
I feel like we should be doing this within the sync method since it's part of the GOV.UK sync, and would then also benefit from retries.

I wonder if we have to ponder the ordering of this too - I'm thinking it might be better that we tell Whitehall we're migrated as the first task in sync, so we don't risk having duplicate items in the search index.

Reply:
Would it perhaps make sense only to tell Whitehall that the document is migrated once the Publishing API publish call has been made?  Otherwise if we tell Whitehall it's migrated prior to that, we could have the situation where the document isn't in any index at all.

Reply:
Yup, that's a risk too. I'd imagine we look less broken just missing the item than having it duplicated. 

Though thinking about it the risk of duplicating is mostly only likely from the subsequent tasks running (clear linkset links and migrate assets) so the sweet spot might be to run this before migrate assets as that's the item with the least consequences for failing.

Reply:
Done 👍